README.rst

```diff
@@ -203,10 +208,22 @@ Scrapers available for:
 - `https://www.heb.com/ <https://www.heb.com/recipe/landing>`_
 - `https://hellofresh.com/ <https://hellofresh.com>`_
 - `https://hellofresh.co.uk/ <https://hellofresh.co.uk>`_
+- `https://www.hellofresh.at/ <https://www.hellofresh.at/>`_
+- `https://www.hellofresh.be/ <https://www.hellofresh.be/>`_
+- `https://www.hellofresh.ca/ <https://www.hellofresh.ca/>`_
+- `https://www.hellofresh.ch/ <https://www.hellofresh.ch/>`_
+- `https://www.hellofresh.co.nz/ <https://www.hellofresh.co.nz/>`_
+- `https://www.hellofresh.com.au/ <https://www.hellofresh.com.au/>`_
 - `https://www.hellofresh.de/ <https://www.hellofresh.de/>`_
+- `https://www.hellofresh.dk/ <https://www.hellofresh.dk/>`_
+- `https://www.hellofresh.es/ <https://www.hellofresh.es/>`_
 - `https://www.hellofresh.fr/ <https://www.hellofresh.fr/>`_
 - `https://www.hellofresh.nl/ <https://www.hellofresh.nl/>`_
 - `https://www.hellofresh.ie/ <https://www.hellofresh.ie/>`_
+- `https://www.hellofresh.it/ <https://www.hellofresh.it/>`_
+- `https://www.hellofresh.lu/ <https://www.hellofresh.lu/>`_
+- `https://www.hellofresh.no/ <https://www.hellofresh.no/>`_
+- `https://www.hellofresh.se/ <https://www.hellofresh.se/>`_
```

Comment:
Agreeing with the existing review comment; this seems repetitive, and I don't think it's a useful way to present the information to readers.

Reply:
So to clarify, what would your desired README look like? Then I can work backwards from that.

Before this PR, the Hello Fresh area of the README looks like this:
> https://hellofresh.com/
https://hellofresh.co.uk/
https://www.hellofresh.de/
https://www.hellofresh.fr/
https://www.hellofresh.nl/
https://www.hellofresh.ie/

Would you just remove all of them except for `hellofresh.com`, and then the test only checks for `hellofresh`, ignoring the TLD?

Reply:
Good question - what do you think of:

* [foo.com.example](https://foo.com.example)
* [bar.com.example](https://bar.com.example) ... providing equivalent support for:
  * [bar.xx.example](https://bar.xx.example)
  * [bar.xy.example](https://bar.xy.example)
* [foo.com.example](https://foo.com.example) ... providing equivalent support for:
  * [qux.com.example](https://qux.com.example)

Reply:
That would work for most existing cases, yep... however, one recent counterexample I can think of is #1060 where compatible content is provided on different domains that vary by more than only the country code.

Reply:
In instances like that example I think it'd be best to just list each site separately, the variants format would only be used in exact domain matches.

Also could be some restrictions on how these formats can be used by what is testable / etc

Reply:
So @jayaddison @jknndy are we happy to proceed with the exact format being:

- hellofresh.com
  - fr, br, com.au, co.uk

With the formal specification being you pick one domain to write fully, then write all the top-level domains that are also supported?

Reply:
That sounds right to me, we should probably add something to the scraper development Docs about this change as well...

Reply:
> So @jayaddison @jknndy are we happy to proceed with the exact format being:
> 
>     * hellofresh.com
>       
>       * fr, br, com.au, co.uk
> 
> 
> With the formal specification being you pick one domain to write fully, then write all the top-level domains that are also supported?

Specifically the output of `host()` (when called without any arguments) should be the container domain, I think.

Reply:
@jknndy does it have to have the full domain listed in the parent, or could it just be the sub-level domain? For example:

- hellofresh
    - .com, .com.au, .uk
    - 
Because otherwise I don't think there is a simple way to extract the TLD from the first example, as you don't know if it is a subdomain or a TLD with a `.` in it (e.g. `.com.au`)

I would just like to raise one more point in favour of having everything listed in its entirety listed out is that I think the main use case for the list is someone using CTRL + F to search for the domain they are interested in. I think that the list is so large that no one will just be casually scrolling through it, and therefore it doesn't matter if it is slightly verbose.

Once again, I know that I am new here so I'm more than happy to continue with your suggestion (in which case, is what I said above okay to do?).

Reply:
Adding a note to mention that I've read this comment, but I haven't yet had time to experiment with the code to check about some ideas I have for answers/suggestions -- and I don't want to reply in full with those until I've figured out whether they're feasible.  I should be able to find time for that within the next day or two.